src/test/article.spec.js

```diff
@@ -213,3 +220,189 @@ describe('GET api/v1/articles', () => {
       });
   });
 });
+describe('POST api/v1/articles/:articleId/report', () => {
+  beforeEach(async () => {
+    await db.User.destroy({ cascade: true, truncate: true });
+    await db.Article.destroy({ cascade: true, truncate: true });
+    await db.Cases.destroy({ cascade: true, truncate: true });
+  });
+  it('should report an article', async () => {
+    const reporter = await createUser();
+    const author = await createUser();
+    const createdArticle = await createArticle(
+      await generateArticle({ authorId: author.id })
+    );
+
+    const createdArticle2 = await createArticle(
+      { id: 2000 },
+      await generateArticle({ authorId: author.id })
+    );
+    const createdReport = await reportArticle(
+      createdArticle.id,
+      await generateReport({
+        userId: reporter.id,
+        articleId: createArticle.id,
+        violation: 'Discrimination'
+      })
+    );
+    const { violation, description } = createdReport[0].dataValues;
+
+    const token = helper.jwtSigner({ id: reporter.id });
+    const res = await server()
+      .post(`/api/v1/articles/${createdArticle2.id}/report`)
+      .set('token', token)
+      .send({ violation, description });
```

Comment:
Good job with the tests 👍, 
However, I suggest `lines 240 - 247` can be converted to `object` that will be sent in on `line 254`.
`Line 248` is not necessary/ideal for this test.
The same suggestion goes for the remaining tests below.

Reply:
it is an object created by the faker, the faker create random test and I am just sending the  variables the represent the random test created by the faker